.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+### Subject of the issue
+
+_Describe your issue here_
+
+### Your environment
+
+- _hardware (past lscpu output)_
+- _operational system name and version_
+
```

Comment:
I would propose the following under the subdirectory `.github/ISSUE_TEMPLATE`, which fits most issues received in the past:

`.github/ISSUE_TEMPLATE/bug_report.md`:
```
---
name: Bug report
about: Report an issue with how PrusaSlicer-ARM.AppImage runs or its packaging
title: ''
labels: ''
assignees: ''

---

**Description of the issue**

**Hardware environment and OS that is being used to run PrusaSlicer-ARM.AppImage**

 - Hardware [e.g. Raspberry Pi 4 (4GB), Raspberry Pi 4 (8GB)]: 
 - OS [e.g. Raspberry Pi OS (32-bit/armhf), Ubuntu Mate 21.04 (64-bit/aarch64)]: 

**Steps to reproduce the reported issue (if applicable)**

**Screenshots (if applicable)**
```

`.github/ISSUE_TEMPLATE/feature_request.md`:
```
---
name: Feature request
about: Suggest a feature that could be implemented in PrusaSlicer-ARM.AppImage
title: ''
labels: ''
assignees: ''

---

**Describe the feature and if there are any other alternatives (if applicable)**

**Does this feature resolve a problem?**

**Additional context (screenshots, diagrams, links to external documentation etc)**
```